test/long/org/apache/cassandra/streaming/LongStreamingTest.java

```diff
@@ -109,6 +109,7 @@ private void testStream(boolean useSstableCompression) throws InvalidRequestExce
         System.err.println(String.format("Writer finished after %d seconds....", TimeUnit.NANOSECONDS.toSeconds(nanoTime() - start)));
 
         File[] dataFiles = dataDir.tryList((dir, name) -> name.endsWith("-Data.db"));
+        //data size is in bytes
```

Comment:
nit: Maybe just call it `dataSizeInBytes`?

Reply:
Good point, I think I left that comment while I was debugging :D I will rename this